.editorconfig

```diff
@@ -10,9 +10,19 @@ insert_final_newline = true
 indent_style = tab
 indent_size = 4
 
-[{*.json,*kml,*.md}]
+[{.*,*.json,*.md}]
 indent_style = space
 indent_size = 2
 
+[site/includes/sitemenureplace.hbs]
+insert_final_newline = false
+
 [*.csv]
 trim_trailing_whitespace = false
+
+[{dep/modernizr-custom.js,src/polyfills/events/mobile.js,src/polyfills/slider/slider.js}]
```

Comment:
Maybe just add a comment above that these are 3rd party scripts

Reply:
Will do :)!